spec/lib/services/subscriptionTest.js

```diff
@@ -0,0 +1,119 @@
+/*
+==============================================================================================
+*/
+
+/*
+var di = require('di'),
+     _ = require('lodash'),
+     core = require('on-core')(di),
+      injector = new di.Injector(
+        _.flatten([
+            core.injectables,
+	    require('./messengerNew.js')
+]));
+	
+messenger = injector.get('Messenger.Test');
+console.log(messenger.messengerFactory);
+//messenger.Messenger.subscribeTimeout();
+*/
+
+/*
+=============================================================================================
+*/
+
+'use strict';
+
+const EventEmitter = require('events');
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'EventEmitter' is defined but never used.